compiler/x/i386/codegen/OMRMachine.hpp

```diff
@@ -64,7 +64,6 @@ class OMR_EXTENSIBLE Machine : public OMR::X86::Machine
       IA32_MAX_GLOBAL_FPRS      = 8,
       };
 
-   TR::RealRegister  *_registerFileStorage[TR_X86_REGISTER_FILE_SIZE];
```

Comment:
Should we deprecate `TR_X86_REGISTER_FILE_SIZE` as well? Will this not always equal `TR::RealRegister::NumRegisters`?